### PR TITLE
Change Password UI

### PR DESCRIPTION
**Pull request by alikarani**

### CODE REVIEW

File: src/containers/ConfirmPasswordContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                        </View>
+                    </View>
+                }
+            ></LayoutComponent>
+        </>
+    );
+}
+const styles = StyleSheet.create({
+    container: {
+        width: '100%',
```
[the remark this answers is not in the document]
Comment (alikarani) [reply src/containers/ConfirmPasswordContainer.tsx:50]: removed

File: src/components/molecules/TPasswordInput.tsx [hunk 2]
```diff
@@ -21,4 +35,20 @@ const styles = StyleSheet.create({
...
+        height: 60,
+        position: 'absolute',
+        right: 0,
+        top: 5,
+        alignItems: 'center',
+        justifyContent: 'center',
+        zIndex: 90000
+    },
+    showHideImage:{
+        tintColor: '#0000008A',
```
[the remark this answers is not in the document]
Comment (alikarani) [reply src/components/molecules/TPasswordInput.tsx:49]: replaced.

File: src/components/molecules/TPasswordInput.tsx [hunk 3]
```diff
@@ -21,4 +35,20 @@ const styles = StyleSheet.create({
         marginBottom: 4,
         borderRadius: 6,
     },
+    showHideContainer: {
```
[the remark this answers is not in the document]
Comment (alikarani) [reply src/components/molecules/TPasswordInput.tsx:38]: fixed.

File: src/navigation/Root.tsx [hunk 4]
```diff
@@ -81,7 +82,7 @@ export default function RootNavigation() {
 
     return (
         <NavigationContainer theme={CombinedDefaultTheme} linking={linking}>
-            <Stack.Navigator initialRouteName="Splash" screenOptions={defaultScreenOptions}>
+            <Stack.Navigator initialRouteName="ConfirmPassword" screenOptions={defaultScreenOptions}>
```
[the remark this answers is not in the document]
Comment (alikarani) [reply src/navigation/Root.tsx]: done

**Comment by alikarani** [comment]
@theblockstalk done with the fixes.

File: src/components/molecules/TPasswordInput.tsx [hunk 5]
```diff
@@ -23,7 +21,7 @@ export default function TPasswordInput(props: TTextInputProps) {
                 style={styles.input}
             />
             <TouchableOpacity style={styles.showHideContainer} onPress={toggleShowHideState}>
-                <Image source={showPassword? closedEyeImage: openEyeImage} style={styles.showHideImage}/>
+                <Ionicons name={showPassword ? 'md-eye' : 'md-eye-off'} size={25} color={customColors.disabledButtonTextColor} />
```
Comment (HaseebUllahAbbasi) [inline src/components/molecules/TPasswordInput.tsx]: linting issues in code, fixing required

**Review by HaseebUllahAbbasi — COMMENTED** [review]
linting issues in code

**Pull request merged**